index.js

```diff
@@ -1321,13 +1323,13 @@ Auth0Lock.prototype._signinPopupNoRedirect = function (connectionName, popupCall
       self._showError(self.options.i18n.t('signin:userConsentFailed'));
     } else if (err.status === 0) {
       self._showError(self.options.i18n.t('networkError'));
-    } else if (err.status !== 401) {
-      self._showError(self.options.i18n.t('signin:serverErrorText'));
-    } else if ('unauthorized' === err.code && err.details && err.details.error_description === 'user is blocked') {
```

Comment:
@nicosabena if we remove this `if` we will no longer handle the case when the user is blocked from the dashboard